src/graphql/search/data/config.ts

```diff
@@ -122,16 +120,6 @@ export const DATA_SEARCH_ENDPOINTS: Array<DataSearchType> = [
     labelSingular: 'Bouw- en omgevingsdossier',
     label: 'Bouw- en omgevingsdossiers',
     searchParam: 'dossier',
-    queryMatcher: /^[A-Z]{2,3}\d/,
-  },
-  {
-    endpoint: SEARCH_ENDPOINTS[DataType.ConstructionFiles],
-    type: DataType.ConstructionFiles,
-    labelSingular: 'Bouw- en omgevingsdossier',
-    label: 'Bouw- en omgevingsdossiers',
-    searchParam: 'olo_liaan_nummer',
```

Comment:
This seems to match a different parameter, that's not an issue?

Reply:
Just found out the `searchParam` doesn't do anything, so I've made it optional